codebuild_specs/scripts/publish.sh

```diff
@@ -16,6 +16,7 @@ else
   git fetch --all
   yarn install
   git restore .
+  yarn production-build
```

Comment:
The `yarn install` is already included in `yarn production-build`. Do we need to remove the duplicate one?

Reply:
Keeping it for now, we can remove it once we verify the fix.